base/nvti.c

```diff
@@ -1203,6 +1219,28 @@ nvti_set_solution_type (nvti_t *n, const gchar *solution_type)
   return 0;
 }
 
+/**
+ * @brief Set the solution method of a NVT.
+ *
+ * @param n The NVT Info structure.
+ *
+ * @param solution_method The solution method to set. A copy will be created
+ *                        from this.
+ *
+ * @return 0 for success. Anything else indicates an error.
+ */
+int
+nvti_set_solution_method (nvti_t *n, const gchar *solution_method)
+{
+  if (!n)
+    return (-1);
+
+  if (n->solution_method)
```

Comment:
There's no need to check this, because g_free does it.

Reply:
Done. Will fix the same for other cases in a separate patch.

Reply:
Thanks.